### PR TITLE
[Component with its own class name as service type] LPS-193924 Refactor notification component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related: https://liferay.atlassian.net/browse/LPS-193924

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:objects

**Comment by Valmir-unicap** [comment]
ci:test:object

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/users/provider/UsersProviderTrackerUtil.java [hunk 1]
```diff
@@ -8,32 +8,29 @@
...
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Feliphe Marinho
  */
-@Component(service = UsersProviderTracker.class)
-public class UsersProviderTracker {
+public class UsersProviderTrackerUtil {
```
Comment (rafaprax) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/users/provider/UsersProviderTrackerUtil.java:18]: Hey @Valmir-unicap, please inline the serviceTracker into its unique usage. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:object

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:object

**Comment by Valmir-unicap** [comment]
I've pushed it now, for rename commit.

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
resent here https://github.com/liferay-core-infra/liferay-portal/pull/5375

**Pull request closed without merging**